src/_base/docker/image/console/Dockerfile.twig

```diff
@@ -7,8 +7,16 @@ RUN chown -R build:build /home/build /app \
  # installing tools in the image is deprecated
  && ([ -e /sbin/tini ] || curl --fail --silent --show-error --location --output /sbin/tini https://github.com/krallin/tini/releases/download/v0.19.0/tini) \
  && chmod +x /sbin/tini \
- && ([ -e /usr/local/bin/mhsendmail ] || curl --fail --silent --show-error --location --output /usr/local/bin/mhsendmail https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64) \
- && chmod +x /usr/local/bin/mhsendmail \
+ && [ -e /usr/bin/msmtp ] || ( \
+    apt-get update -qq \
+    && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install \
+    # package dependencies \
+      msmtp \
+    # clean \
+    && apt-get auto-remove -qq -y \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+ ) \
```

Comment:
since the previous lines were only for BC, I don't think we need this installed here as it'd be in the upstream image

Reply:
Okay let's merge https://github.com/my127/docker-php/pull/55 instead and I'll update this PR to remove it :+1:

Reply:
Updated